Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -597,5 +597,161 @@
         }
       }
     ]
+  },
+  {
+    "Input": "organizame una reunion con Francesco Sparacio mañana a las 12",
+    "Context": {
+      "ReferenceDateTime": "2020-01-23T01:00:00"
+    },
+    "NotSupported": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "una ",
```

Comment:
This is incorrect. Can you take a look at the examples provided in the issue for the cases where "una" should be time?
Isn't this similar to what you did in French?